pkgs/by-name/xf/xfox/package.nix

```
@@ -0,0 +1,32 @@
+{
+  python3Packages,
+  fetchFromGitHub,
+  lib,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "xfox";
+  version = "1.0.0";
+  pyproject = true;
+  src = fetchFromGitHub {
+    owner = "pxlman";
+    repo = "xfox";
+    tag = "v${version}";
+    sha256 = "sha256-gbcJ02ZaPqiCo0VHnVOYzrWR00dQXgKavFTgP+6rS34=";
+  };
+  propagatedBuildInputs = [
+    python3Packages.evdev
+    python3Packages.libevdev
+    python3Packages.setuptools
+  ];
+  postPatch = ''
+    substituteInPlace setup.py --replace-fail "evdev-binary" "evdev"
+  '';
+  meta = {
+    description = "Tool to remap gamepads to act as a virtual xbox gamepad.";
+    homepage = "https://github.com/pxlman/xfox";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [
+      pxlman
+    ];
+  };
```

And mainProgram should be there too, as `xfox` is the only executable
```suggestion
    mainProgram = "xfox";
  };
```